config.libsonnet

```diff
@@ -67,6 +67,8 @@
     // allow you to overcommit if you wish.
     namespaceOvercommitFactor: 1.5,
     kubeletPodLimit: 110,
+    kubeletPlegIntervalWarningSeconds: 5,
```

Comment:
Note: I'm unsure of these thresholds... they're aligned well against the clusters I have access to and would fire alerts appropriately, but I lack any insight to clusters running high number of pods (or anything else that may affect the PLEG interval).